src/layers.js

```diff
@@ -153,7 +184,10 @@ const transformBounds = (matrices, { x1, y1, x2, y2 }) => {
   };
 };
 
-const transformPoints = (matrices, points) =>
+const transformPoints = (
+  matrices: Array<TypeMatrix>,
+  points: Array<Array<number>>
+): Array<Array<number>> =>
```

Comment:
Flow let's you describe tuples, which would be a more accurate type for the points:

```js
Array<[number, number]>
```